static/src/javascripts/projects/commercial/modules/third-party-tags/outbrain-tracking.js

```diff
@@ -2,10 +2,10 @@
 
 import ophan from 'ophan/ng';
 
-const tracking = function(trackingObj: {
-    widgetId?: string,
+const tracking = (trackingObj: {
+    widgetId?: ?string,
```

Comment:
Since this field is already optional, I don't think it makes sense for it to be nullable as well. Instead of explicitly setting this to `null`, why don't we simply omit the property altogether? Does tracker-js (or Ophan itself) behave differently depending on whether this property is `null` or `undefined`?